search-app/test/cmr/search/test/api/tags.clj

```diff
@@ -0,0 +1,38 @@
+(ns cmr.search.test.api.tags
+  "Tests to verify functionality in cmr.search.api.tags namespace."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.search.api.tags :as tags]))
+
+(def dictionary-with-error [
```

Comment:
Nitpick with both `def`s: the object should only be indented 2 spaces. Not a big deal, but it's our convention. Let me know if you can't find an example and I'd be happy to help